drag_and_drop_v2/public/js/view.js

```diff
@@ -50,13 +51,21 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
+                            // so remove the ability to move focus to it using the keyboard
         }
         return (
             h('div.option',
                 {
                     key: item.value,
                     className: item.class_name,
-                    attributes: {'data-value': item.value, 'data-drag-disabled': item.drag_disabled},
+                    attributes: {
+                        'tabindex': tabindex,
+                        'draggable': true,
+                        'aria-grabbed': false,
```

Comment:
@mtyaka What you are suggesting won't work with `state.items[item_id]` and `applyState` because

* the item being dragged only gets added to `state.items` *after* being dropped (which makes sense considering that `state.items` keeps track of items that have already been placed)
* `applyState` will destroy the draggable mid-drag (by calling `destroyDraggable`)

I pushed a version that uses `configuration.items` and `updateDOM` instead.